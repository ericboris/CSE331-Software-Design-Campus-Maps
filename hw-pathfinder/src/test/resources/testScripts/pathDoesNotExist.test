# Path doesn't exist

CreateGraph g

AddNode g A
AddNode g B

FindPath g A B
