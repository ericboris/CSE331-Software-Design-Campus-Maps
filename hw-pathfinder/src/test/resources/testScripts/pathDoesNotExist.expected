# Path doesn't exist

created graph g

added node A to g
added node B to g

path from A to B:
total cost: 0.000